### ISSUE TITLE
Website design needs

### ISSUE THREAD

**Issue by migurski** [openaddresses/openaddresses-ops#15]
There are a few ops issues that in some way connect with the design of the OA site: a place to [post coverage data](https://github.com/openaddresses/openaddresses-ops/issues/14), a [generated list of source attributions](https://github.com/openaddresses/openaddresses.io/pull/67), an eventual [long-term home for parcels](https://github.com/openaddresses/machine/milestone/2), and the basic visual dissonance between https://openaddresses.io and https://results.openaddresses.io.

In the past Mapbox has donated web design/development resources to get OA’s site into shape. @ingalls believes this might be possible again, and asked that we put something together that would help scope the need. At Code for America and now at Mapzen, I’ve commissioned web style guides and found them to be enormously helpful in visually unifying a spread-out collection of web resources. This issue is an attempt to describe a style guide that we hope Mapbox might donate to build on their previous work.

These are some requirements we used when creating the [Mapzen styleguide](https://mapzen.com/common/styleguide/):

1. Styleguide must make the creation of on-brand, correct-looking web sites easy for all collaborators.
2. It needs to be usable by URL and by copy/paste, according to easy-to-follow directions.
    * URLs should point to simple, plain CSS and JS resources.
    * The preferred interaction is copy/paste from example pages and pattern snippets.
    * No direct use of pre-processor sources or raw includes.
3. Samples should exist for a couple common page types. Right now those include a big-header front page and a small-header content page.
4. Patterns should exist for lots of common site elements.    Basic text markup such as headers, lists, paragraphs, and so on should Just Work and might not need any special mention in the styleguide.

Is anything from this description clearly missing? Is this a reasonable request?

**Comment by heyaustinwood** [comment]
@migurski I've been watching the oa project for quite some time, as we're planning on using address data in our new project this year at Neighborhoody.

Looking through the open issues, this one caught my eye. I'm not an engineer nor do I have a gis background, so I haven't found a place to contribute to the project with my skillset. I come from the creative world, not engineering, with a background in design, web-dev, and ui/ux. 

This issue is still open, and recently @ingalls removed his assignment, so... are you looking for contributors for a web style guide?

**Comment by ingalls** [comment]
@heyaustinwood This would be amazing if you have time!

My plan was to migrate from mapbox/base (What the current website is based on) to use: https://www.mapbox.com/assembly/

That said, I'm open to suggestions and ideas for getting to a more documented system so we can make results.openaddresses.io more stylistically similar to the main page

Happy to answer questions if you want to take this on

**Comment by heyaustinwood** [comment]
@ingalls great. Available for a quick call sometime this week?

**Comment by ingalls** [comment]
@heyaustinwood yup for sure, send me an email at ingalls@protonmail.com and we can exchange phone numbers

**Comment by migurski** [comment]
Excited you two are looking at this! Nice to meet you Austin, glad you’re interested in helping.
